test/jit/test_upgraders.py

```diff
@@ -26,8 +26,12 @@ def _load_model_version(self, loaded_model):
         torch.jit.save(loaded_model, buffer)
         buffer.seek(0)
         zipped_model = zipfile.ZipFile(buffer)
-        version = int(zipped_model.read('archive/version').decode("utf-8"))
-        return version
+        try:
```

Comment:
Add a comment for what's going on here and why there's a fallback

Reply:
Sounds good.